stages/org.osbuild.dmverity.meta.json

```diff
@@ -0,0 +1,42 @@
+{
+  "summary": "Enables dm-verity protection",
+  "description": [
+    "Sets up dm-verity for data_device and stores hash blockes on hash_device.",
+    "Root hash gets written to `root_hash_file`"
+  ],
+  "schema_2": {
+    "options": {
+      "additionalProperties": false,
+      "required": [
+        "root_hash_file"
+      ],
+      "properties": {
+        "blocksize": {
+          "type": "number",
+          "default": 512
+        },
+        "root_hash_file": {
+          "type": "string"
+        }
+      }
+    },
+    "devices": {
+      "type": "object",
+      "additionalProperties": true,
+      "required": [
+        "data_device",
+        "hash_device"
+      ],
+      "properties": {
+        "data_device": {
+          "type": "object",
+          "additionalProperties": true
+        },
+        "hash_device": {
+          "type": "object",
+          "additionalProperties": true
+        }
```

Comment:
The schema marks `data_device` and `hash_device` as required, but not the `path` subkey.  Instead, the schemas for the `device` objects is empty with just `additionalProperties: true`.  I assume the paths are required and don't have fallback values, so the schema should reflect that:
```suggestion
        "data_device": {
          "type": "object",
          "additionalProperties": false,
          "required": ["path"],
          "properties": {
            "path": {
              "type": "string"
            }
          }
        },
        "hash_device": {
          "type": "object",
          "additionalProperties": false,
          "required": ["path"],
          "properties": {
            "path": {
              "type": "string"
            }
          }
        }
```